draft-birkholz-scitt-software-supply-chain-use-cases.md

```diff
@@ -78,6 +99,7 @@ deployment chain diagram here
 {: #deployment-chain title="Deployment Example of SCITT in Software Development"}
 
 # Software Supply Chain Use Cases
+SCITT fundamental architecture emphasizes heavily on introducing transparency and accountability to the entire steps involved in software delivery. With the advancement of Software Bill of Materials (SBOM) as a standard practice for managing software deliveries the below use case will take SBOM as an example to illustrate how a SCITT based system can augment security built using SBOMs.
```

Comment:
This reads as if only one SBOM use case follows, but there are three software supply chain use cases?